### PR TITLE
Add a TensorFlow Example

### PR DESCRIPTION
**Pull request by aallan**
TensorFlow Lite inferferencing examples that carry out real-time object detection and labelling in a QT preview window using the [MobileNet V2](https://arxiv.org/abs/1801.04381) network trained on the [COCO dataset](https://cocodataset.org/#home). Still needs optimising but is good enough for an initial check-in!

### CODE REVIEW

File: examples/tensorflow/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# TensorFlow Lite Example
+
+TensorFlow Lite inferferencing examples that carry out real-time object detection and labelling in a QT preview window using the [MobileNet V2](https://arxiv.org/abs/1801.04381) network trained on the [COCO dataset](https://cocodataset.org/#home).
```
Comment (andrum99) [inline examples/tensorflow/README.md:3]: `inferferencing` -> `inferencing`

File: examples/tensorflow/README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+## Installing Picamera 2
```
Comment (andrum99) [inline examples/tensorflow/README.md:5]: Remove the space between `Picamera` and `2`.

File: examples/tensorflow/README.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+Ensure that you have installed the Picamera 2 library along with OpenCV.
```
Comment (andrum99) [inline examples/tensorflow/README.md:7]: And again here.

File: examples/tensorflow/real_time.py [hunk 4]
```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/python3
+
+# Copyright (c) 2022 Raspberry Pi Ltd
+# Author: Alasdair Allan <alasdair@raspberrypi.com>
+# SPDX-License-Identifier: BSD-3-Clause
+
+# A TensorFlow Lite example for Picamera2 on Raspberry Pi OS Bullseye
+#
+# Install necessary dependences before starting,
```
Comment (andrum99) [inline examples/tensorflow/real_time.py:9]: Extraneous comma at end of line

File: examples/tensorflow/real_time_with_labels.py [hunk 5]
```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/python3
+
+# Copyright (c) 2022 Raspberry Pi Ltd
+# Author: Alasdair Allan <alasdair@raspberrypi.com>
+# SPDX-License-Identifier: BSD-3-Clause
+
+# A TensorFlow Lite example for Picamera2 on Raspberry Pi OS Bullseye
+#
+# Install necessary dependences before starting,
```
Comment (andrum99) [inline examples/tensorflow/real_time_with_labels.py:9]: Another rogue comma

File: examples/tensorflow/real_time.py (hunk 4 above)
Comment (aallan) [reply examples/tensorflow/real_time.py:9]: it's not extraneous it's just a single run on sentence.

**Comment by aallan** [comment]
Every so often "with labels" example crashes with an "index out of bounds" error caused by a race condition as the rectangles and texts arrays are updated by TensorFlow before this loop finishes. This doesn't happen for the non-indexed version without labels. Solution will be to rework the loop, append the label to the rectangles array rather than use a separate array. I'll fix the bug later today and repush.

File: examples/tensorflow/real_time_with_labels.py (hunk 5 above)
Comment (aallan) [reply examples/tensorflow/real_time_with_labels.py:9]: Not rogue, intended.

**Pull request merged**